File: admin/src/main/java/hr/nas2skupa/eleventhhour/admin/ProviderViewHolder.java
package hr.nas2skupa.eleventhhour.admin;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.TextView;

import hr.nas2skupa.eleventhhour.common.model.Provider;

/**
 * Created by nas2skupa on 28/02/2017.
 */

public class ProviderViewHolder extends RecyclerView.ViewHolder {
    private TextView titleView;


    public ProviderViewHolder(View itemView) {
        super(itemView);

        titleView = (TextView) itemView.findViewById(R.id.txt_provider_name);
    }

    public void bindToProvider(Provider provider) {
        titleView.setText(provider.name);
    }
}

File: admin/src/main/java/hr/nas2skupa/eleventhhour/admin/viewholders/ProviderViewHolder.kt
package hr.nas2skupa.eleventhhour.admin.viewholders

import android.support.v7.widget.RecyclerView
import android.view.View
import hr.nas2skupa.eleventhhour.common.model.Provider
import kotlinx.android.synthetic.main.item_provider.view.*

/**
 * Created by nas2skupa on 28/02/2017.
 */

class ProviderViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    fun bind(provider: Provider) = with(itemView) {
        txt_provider_name.text = provider.name
    }
}